### PR TITLE
Adding Static Code Analysis

### PR DESCRIPTION
**Pull request by Daniela8699**

### CODE REVIEW

**Comment by Daniela8699** [comment]
Estas fallando los checks, pero no entiendo por que?

File: package.json [hunk 1]
```diff
@@ -3,40 +3,42 @@
...
-    "test:headless": "npm run build && protractor dist/protractor/headless.config.js"
+    "test:headless": "npm run build && protractor dist/protractor/headless.config.js",
+    "test": "npm run test:headless",
+    "postinstall": "webdriver-manager update --gecko false",
+    "lint": "tslint --project tsconfig.json protractor/**/*.ts test/**/*.ts src/**/*.ts"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Daniela8699/protractor-workshop-2020.git"
+    "url": "git+https://github.com/manuelq12/protractor-workshop-2020.git"
```
Comment (Valeryibarra) [inline package.json:30]: 🤔

File: package.json [hunk 2]
```diff
@@ -3,40 +3,42 @@
...
   },
   "keywords": [
-    "\u0016ui-testing",
+    "ui-testing",
     "dojo",
     "practice",
     "protractor"
   ],
-  "author": "Daniela Llano <daniela.llano1@correo.icesi.edu.co> (Daniela8699)",
+  "author": "[Manuel Quintero] <quinteromanuel1202@gmail.com> (manuelq12)",
```
Comment (Valeryibarra) [inline package.json:38]: aqui también

File: package.json [hunk 3]
```diff
@@ -3,40 +3,42 @@
...
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/Daniela8699/protractor-workshop-2020/issues"
+    "url": "https://github.com/manuelq12/protractor-workshop-2020/issues"
```
Comment (Valeryibarra) [inline package.json:41]: :arrow_up:

File: package.json [hunk 4]
```diff
@@ -3,40 +3,42 @@
...
-  },
-  "devDependencies": {
-    "@types/jasminewd2": "^2.0.8",
-    "@types/node": "^14.0.25",
-    "protractor": "^7.0.0",
-    "jasmine-spec-reporter": "^5.0.2",
-    "typescript": "^3.9.7"
-
-  }
+  "homepage": "https://github.com/manuelq12/protractor-workshop-2020#readme"
```
Comment (Valeryibarra) [inline package.json:43]: :arrow_up:

**Comment by Daniela8699** [comment]
> there are other PRs open for your repo, try first to finish one by one in order to use the changes of the previous branch approved

Ok,thank you but I need to close the other PR?

**Comment by Valeryibarra** [comment]
keep this one open is the next according to your previous changes, I already put the comments of this PR.
close the other one "done with the debug code" 
After this PR is approved you can merge this branch (into master) and close (this branch), after that you should create a new branch from master. Master will have all changes approved in this PR, therefore the branch that you create from master too, so the idea is to do the steps sequentially

**Pull request merged**